InnoTopicWebsite/src/app/work-projects/work-projects-data.ts

```diff
@@ -131,7 +131,7 @@ export class WorkOrganisationsAndProjectsData {
       'EpicElite Platform': project({
         roles: 'Lead/Senior Full-Stack Developer & Interviewing and Recruiting (Consultant)',
         description: 'Recruited and interviewed 5 Developers and a UI/UX/Web/Graphics Designer. I have lead a team of 5 Developers. Did code&architecture reviews.' +
-          'Working directly with CEO & Director of Operations. Communication in English, Spanish, Polish. #Python #Django + #TypeScript app using #MongoDB and #MariaDB .',
+          'Working directly with CEO & Director of Operations. Communication in English, Spanish, Polish.',
```

Comment:
@samyakjainnnnn above hashtags were placed there on purpose so I guess they should stay there